pkg/virt-launcher/virtwrap/network/podinterface_test.go

```diff
@@ -586,6 +586,14 @@ var _ = Describe("Pod Network", func() {
 			})
 		})
 		Context("Macvtap plug", func() {
+			BeforeEach(func() {
+				primaryPodInterface.HardwareAddr = fakeMac
```

Comment:
Why here and not on the interface creation (same as the name and the mtu)?

Reply:
Because I don't want all the tests to use a `primaryPodInterface` having a MAC (and update the unit tests to account for it - or fix their bind mechanism implementations). 

Any of that would be nice to have, but, should happen on separate PRs. This one's scope if quite clear.

Reply:
I've had no reply, so just went ahead and did it. 

Is it better now ? @AlonaKaplan